addons/account/models/account_journal_dashboard.py

```diff
@@ -554,10 +548,10 @@ def _fill_sale_purchase_dashboard_data(self, dashboard_data):
             (number_waiting, sum_waiting) = self._count_results_and_sum_amounts(query_results_to_pay[journal.id], currency)
             (number_draft, sum_draft) = self._count_results_and_sum_amounts(query_results_drafts[journal.id], currency)
             (number_late, sum_late) = self._count_results_and_sum_amounts(late_query_results[journal.id], currency)
-            amount_total_signed_sum, count = to_check_vals.get(journal.id, (0, 0))
+            (number_to_check, sum_to_check) = self._count_results_and_sum_amounts(to_check_vals[journal.id], currency)
```

Comment:
You're probably want to stay consistent but in case you didn't know, parentheses are not necessary here

Reply:
Yep, I know but it felt weird to not do it with the above lines :D